### DISCUSSION TITLE
Please add "Discussions" in the site navbar

### DISCUSSION THREAD

**Discussion by expikr** [community/community#21220] [Discussions]
![image](https://user-images.githubusercontent.com/77922942/180694042-bf0c89ef-4b0c-4764-aa42-9f12ce23ec53.png)

I use Discussions and Issues often as my "self-conversation" notebook, like how some people create a Discord server to dump random thoughts/attachments, I find GitHub's ability to create repositories as self-contained "servers" to be far superior and with better UX, especially with the addition of Math equation support.

I would like to be able to quickly click on a shortcut in the navbar rather than having to open the dropdown and visually search for the right item to click on.

**Comment by ghost** [comment]
i think its useful Idea

**Comment by jfullstackdev** [comment]
by now it's there

![image](https://user-images.githubusercontent.com/47092464/181004314-5a0c8b99-dd94-4748-8c31-800edeb8622f.png)

what you can do is to simply bookmark the URL, so it's always one click
